hazelcast/src/main/java/com/hazelcast/cache/impl/nearcache/impl/store/NearCacheObjectRecordStore.java

```diff
@@ -137,4 +144,18 @@ public int size() {
         return store.size();
     }
 
+    @Override
```

Comment:
data record store and object record store classes looks very similar, can't you use a base class to eliminate code duplication ?